extensions/ibox/src/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState, useEffect } from "react";
+import fetch from "node-fetch";
+import { ActionPanel, Detail, List, Action } from "@raycast/api";
+import { ResData, BaseListData, Item, AsyncStatus } from "./type";
+import { useAsync } from "./hooks";
+
+export default function Command() {
+  const [searchText, setSearchText] = useState("");
+
+  const { status, value, reset, execute } = useAsync<BaseListData<Item>>(fetchResource);
+
+  useEffect(() => {
+    console.log(`searchText: ${searchText}`);
+    if (!searchText) {
+      reset();
+      return;
+    }
+
+    execute(searchText);
+  }, [searchText]);
+
+  // searchText={searchText} 导致无法输入中文
+  return (
+    <List
+      navigationTitle="搜索书签"
```

Comment:
```suggestion
```
Don't change the `navigationTitle` in the root command - it will be automatically set to the command name. Use `navigationTitle` only in nested screens to provide additional context.

Reply:
done